enhancements/testing/ephemeral-agent-installer-e2e-tests.md

```diff
@@ -0,0 +1,275 @@
+---
+title: ephemeral-agent-installer-e2e-tests
+authors:
+  - "@lranjbar"
+reviewers:
+  - "@rwsu"
+  - "@zaneb"
+  - "@pawanpinjarkar"
+approvers:
+  - "@dhellmann"
+  - "@celebdor"
+api-approvers: # in case of new or modified APIs or API extensions (CRD, aggregated api servers, webhooks, finalizers)
+  - N/A
+creation-date: 2022-02-10
+last-updated: 2022-02-15
+tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
+  - https://issues.redhat.com/browse/AGENT-20
+---
+
+# Ephemeral Agent Installer e2e Tests
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Create a new e2e testing framework for the new agent based installer project. 
+Currently, for this project there would be potentially two existing e2e frameworks 
+that could be used, dev-scripts and assisted-test-infra. However, both of these have 
+significant challenges in re-purposing them for this new project. 
+
+The main challenge with either of these existing frameworks is that the environment
```

Comment:
On a purely formatting note, it's nice to keep the "summary" section short (~1 paragraph). I think your first paragraph here is OK for that, and the rest of this section could move down to either the "motivation" or "proposal" sections.